client/src/Components/Students.js

```diff
@@ -32,8 +24,8 @@ const Students = (props) => {
 			<input
 				type="text"
 				onChange={changeHandler}
-				placeholder="Enter the code"
-				className="input"
+				placeholder="Enter quiz code"
+				className="studentCodeinput"
```

Comment:
why did you change the class name, I just try to prevent of repeatative code and defined a class name to use it for all text input

Reply:
you could be right here, i did not know you use it and wanted to align input differently probably adding another class for alignment can be a better solution